instrumentation/src/javatest/org/oppia/android/instrumentation/ExplorationPlayerTest.kt

```diff
@@ -0,0 +1,108 @@
+package org.oppia.android.instrumentation
+
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.UiCollection
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiScrollable
+import androidx.test.uiautomator.UiSelector
+import androidx.test.uiautomator.Until.hasObject
+import org.junit.Assert.assertNotNull
+import org.junit.Before
+import org.junit.Test
+
+/** Tests for Explorations. */
+class ExplorationPlayerTest {
+  private val OPPIA_PACKAGE = "org.oppia.android"
+  private val LAUNCH_TIMEOUT = 30000L
+  private val TRANSITION_TIMEOUT = 5000L
+  private lateinit var device: UiDevice
+
+  @Before
+  fun startOppiaFromHomeScreen() {
+    // Initialize UiDevice instance
+    device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+
+    // Start from the home screen
+    device.pressHome()
+
+    // Wait for launcher
+    val launcherPackage = getLauncherPackageName()
+    assertNotNull(launcherPackage)
+    device.wait(hasObject(By.pkg(launcherPackage)), LAUNCH_TIMEOUT)
+
+    // Launch the blueprint app
+    val context = ApplicationProvider.getApplicationContext<Context>()
+    val intent = context.packageManager
+      .getLaunchIntentForPackage(OPPIA_PACKAGE)
```

Comment:
```suggestion
    val intent = context.packageManager.getLaunchIntentForPackage(OPPIA_PACKAGE)
```

This can fit on one line.

Reply:
Done